RPIPs/RPIP-committee_stipends.md

```diff
@@ -0,0 +1,130 @@
+* The Global Stipend Rate SHALL be modified only by the pDAO via a voted modification to this RPIP and SHOULD follow the usual processes for such a modification. 
+
+### Current and Historic Parameter Values
+
+| Date                       | Global Stipend Rate |
+|----------------------------|---------------------|
+| 2024-03-04 (RPIP Authored) | 30 USD / hr         |
+
+#### Grants Management Committee
+
+| Date                       | Target Member Count | Target Member Hours | Total Monthly Stipend Budget |
+|----------------------------|---------------------|---------------------|------------------------------|
+| 2024-03-04 (RPIP Authored) |                   9 |                  13 |                    $3,510.00 |
+
+#### Incentives Management Committee
+
+| Date                       | Target Member Count | Target Member Hours | Total Monthly Stipend Budget |
+|----------------------------|---------------------|---------------------|------------------------------|
+| 2024-03-04 (RPIP Authored) |                   9 |                  13 |                    $3,510.00 |
+
+
+<!--
+Commentary for Editors:
+The above tables may be generated using this [google sheet](https://docs.google.com/spreadsheets/d/1aXE-bbVK5IB54TfqRPq9_kyMKMDDDiVDuw3yA3f5XJg/edit#gid=958081945), which should be maintained in parallel. If you need to modify the sheet and do not have permission then:
+1. Copy the sheet
+2. Set it to public read-only
+3. Replace this link with your updated sheet
+4. Convert the new sheet to markdown using: https://www.tablesgenerator.com/markdown_tables or similar.
+-->
+
+### Payments
+
+#### Split
+The split of the committee stipend budget SHOULD be determined by the voting members of the committee in question. 
+* Each committee SHOULD be consistent in the method used to determine the payout split. 
+* Roles that are compensated through other mechanisms SHOULD be discounted when the committee determines the stipend split.
+
+#### Distribution
+The Grants Management Committee is responsible for distributing the stipends for each committee directly to its members.
+* Stipends SHOULD be paid out in one of the GMC's accepted payout token options, selected by the recipient.
+  * Conversion from USD to dollar-stablecoin payout token SHOULD be priced 1:1. 
+  * Conversion from USD to any other payout token SHOULD be done using the market price at the time of payment transaction creation. 
+* Other committees MUST provide the GMC Administrator with the details of their stipend split each month. 
+  * They MUST communicate payment addresses, stipend share in USD, and requested payment token for each of their members. 
+  * They MUST meet any deadline or format requirements imposed by the GMC Administrator.
+* Any undistributed funds for a given month are forfeited by that committee.
```

Comment:
This seems un-necessarily punitive. If it becomes a problem, I'm fine with adding it, but if I currently had to guess why such a thing would happen it's like "people being busy with a crypto conference" or something. Ie, we might piss people off or make them choose to attend to the urgent over the important.

Reply:
So, I think this is actually fairly important. If there are not hard deadlines + consequences for missing them, then the burden to 'catch-up' ends up on the GMC admin. I had a lot of experience at this at Maker, and it sucks to be in those shoes and have to decide whether to be punitive to others or just accept the cost to yourself. 

Given that the deadline is set by the GMC admin, they can adjust said deadline if there is a conference or such coming up. I'd expect them to provide common courtesy and adjust the deadline if it was generally agreed to be necessary.

Reply:
Hmmm... fair... the fact that it's a human doing the imposing should provide a lot of wiggle room. Convinced enough.